### DISCUSSION TITLE
How to deselect ebsd data following a contour?

### DISCUSSION THREAD

**Discussion by NovakEvana** [mtex-toolbox/mtex#1760] [Ask Anything] [answered]
Hi everyone 

My scanned EBSD result consisted of both metal and mounting resin material. How to deselect (remove) data of resin material from my result following a specified contour (shape)? The data I'm mentioning here means ebsd and grain boundaries information. The current IQ, phase, and GOS map below still count them in.

Thank you,
Evana.

**Comment by NovakEvana** [comment]
Please let me know if anyone has any solution for this.  Thank you so much in advance!

Evana.

**Comment by kilir** [comment, marked as answer]
Hi,
EBSD/inpolygon() will do this.
Cheers,
Rüdiger

**Reply by NovakEvana** [reply]
Thanks Rudiger!

**Reply by huseyinhzl** [reply]
Hi Avena,

I am having the same issue and I couldn't solve it until now, so can I ask you how you implemented the EBSD/inpolygon command?

Thank you

**Reply by NovakEvana** [reply]
Hi, you can first generate a phase map. Then click on the phase map to pick points following the contour you want to generate corresponding EBSD data. Then input your retrieve x,y coordinates using the command [inpolygon](https://mtex-toolbox.github.io/EBSD.inpolygon.html).

Evana

**Reply by huseyinhzl** [reply]
Thank you so much. I was trying some other methods which were giving strange data sets. This solution will be better. Thank you again
